_reports/ksdme.md

```diff
@@ -0,0 +1,403 @@
+
+  - 16:
+      - repo: coala-ls
+      - link: https://github.com/coala/coala-ls/pull/8/commits/cee8c2c7c047999d639b03802e0e3c0f029e169e
+      - details: >
+         README: Update README.md and assets
+
+  - 17:
+      - repo: coala-ls
+      - link: https://github.com/coala/coala-ls/pull/9/commits/ef8dc592c80d629d90c0596bb1a2be3df75c7b40
+      - details: >
+         .coafile: Update config and globs
+
+  - 18:
+      - repo: coala-ls
+      - link: https://github.com/coala/coala-ls/pull/11/commits/8d7f8d05e506b252179c92335f98cb365102a405
+      - details: >
+         perf-metrics.md: Add performance metrics
+
+  - 19:
+      - repo: coala-ls
+      - link: https://github.com/coala/coala-ls/pull/13/commits/32fe61caa96d8a039cb532bb46eb1ed8e10cbfb4
+      - details: >
+          coalals.results: Refactor to submodule
+
+  - 20:
+    - repo: coala-ls
+    - link: https://github.com/coala/coala-ls/pull/13/commits/e9745d34b9705787bad41df12a1801eee4cfab7e
+    - details: >
+        requirements.txt: coalals.utils.files: Add get_disk_contents()
+
+  - 21:
+      - repo: coala-ls
+      - link: https://github.com/coala/coala-ls/pull/13/commits/88d79837106185382d3feadc8c9330f219ea271e
+      - details: >
+         requirements.txt: Add whatthepatch
+
+  - 22:
+      - repo: coala-ls
+      - link: https://github.com/coala/coala-ls/pull/13/commits/e9745d34b9705787bad41df12a1801eee4cfab7e
+      - details: >
+         coalals.results.fixes: Add fixes parsing & lsp models
+
+  - 23:
+      - repo: coala-ls
+      - link: https://github.com/coala/coala-ls/pull/13/commits/7b4d59d0422be7017dc5fa14a4a7879bc5361c63
+      - details: >
+         coalals.results.diagnostics: Support fixes collection
+
+  - 24:
+      - repo: coala-ls
+      - link: https://github.com/coala/coala-ls/pull/13/commits/8e75ead740fac57b461bb12497db4147f78d4d19
+      - details: >
+         coalals.langserver: Add support for textDocument/formatting
+
+  - 25:
+      - repo: coala-ls
+      - link: https://github.com/coala/coala-ls/pull/14/commits/99c0663b0177bba30a4b8025615530d15e4d441b
+      - details: >
+         coalals.langserver: Add support for showMessageRequest
+
+  - 26:
+      - repo: coala-ls
+      - link: https://github.com/coala/coala-ls/pull/13/commits/7b4d59d0422be7017dc5fa14a4a7879bc5361c63
+      - details: >
+         coalals.results.diagnostics: Support fixes collection
+
+  - 27:
+      - repo: coala-ls
+      - link: https://github.com/coala/coala-ls/pull/13/commits/7b4d59d0422be7017dc5fa14a4a7879bc5361c63
+      - details: >
+         coalals.results.diagnostics: Support fixes collection
+
+  - 28:
+      - repo: coala-ls
+      - link: https://github.com/coala/coala-ls/pull/15/commits/6e095ca6c18813878766e9767d99a4bec10ec4a8
+      - details: >
+         perf-metrics.md: Update perf-metrics for Phase-2
+
+  - 29:
+      - repo: coala-vs-code
+      - link: https://github.com/coala/coala-vs-code/pull/73/commits/52319c23e653850df148b284882fce799724c32c
```

Comment:
blocked on https://github.com/coala/coala-vs-code/pull/73 getting merged  -- those commit references will disappear on a rebase, so you can not use them.